cuda/private/os_helpers.bzl

```diff
@@ -35,3 +35,31 @@ def cc_import_versioned_sos(name, shared_library):
         name = name,
         deps = [":%s" % paths.basename(p) for p in so_paths],
     )
+
+def win_share_library_provided_runtime(name, system_provided = True):
+    """Creates a cc_library that depends on all versioned .so files with the given prefix.
+
+    If <shared_library> is path/to/foo.so, and it is a symlink to foo.so.<version>,
```

Comment:
What is the doc for?

Reply:
it's error doc

win_share_library_provided_runtime mecro is to help find windows shared library runtime file path 
(if specaified cuda_version or specaified  CUDA_PATH , it need to get right runtime file to bazel test  )